libcontainer/process_linux.go

```diff
@@ -836,6 +840,28 @@ func (p *initProcess) createNetworkInterfaces() error {
 	return nil
 }
 
+// setupNetworkDevices sets up and initializes any defined network interface inside the container.
+func (p *initProcess) setupNetworkDevices() error {
+	// host network pods does not move network devices.
+	if !p.config.Config.Namespaces.Contains(configs.NEWNET) {
+		return nil
```

Comment:
I think you already fail in the config validate if there is no netns. 

This _might_ be useful for libct users, but for runc this should never happen.

Reply:
should I remove it then?

Reply:
I think both are fine, let's keep it? No strong opinion.